frontend/public/locales/en/apply-adult-child.json

```
@@ -219,6 +219,9 @@
     "register-msca-correspondence": "Official correspondence from the Government of Canada",
     "register-msca-confirm": "Letters confirming acceptance or rejection of your application",
     "register-msca-checker": "While we work on making these features available, we encourage you to use the <mscaLinkChecker>Canadian Dental Care Plan Status Checker</mscaLinkChecker> to stay updated on your application's progress.",
+    "register-msca-title-featured": "Register for a My Service Canada Account (MSCA)",
+    "register-msca-text-featured": "Get important updates about your Canadian Dental Care Plan application by registering for a<mscaLinkAccount>My Service Canada Account</mscaLinkAccount>.",
```

```suggestion
    "register-msca-text-featured": "Get important updates about your Canadian Dental Care Plan application by registering for a <mscaLinkAccount>My Service Canada Account</mscaLinkAccount>.",
```

(same comment for the other English i18n JSON files)